### ISSUE TITLE
Is there a good cross-language package manager ?

### ISSUE THREAD

**Issue by rom1504** [ProtoDef-io/ProtoDef#7]
Currently we're package this (and minecraft-data) with submodules. It's meh.

Is there a good cross-language package manager we could use as an alternative ?

I guess there's nix but I'm not sure if it applies.

Not expecting to solve that issue very quickly, but I'd like to find something good for this eventually.

A good first step might be to do git tag and tell people in the readme to depend on the commit of the tags instead of random commits.

**Comment by roblabla** [comment]
git is the cross-language package manager. `nix` is only good for people who go all-in on it.

What is this repo packaging ? Only the tests right ?

**Comment by rom1504** [comment]
Yes it only package tests but tests are kind of equivalent to the spec. 
Being able to look at a history.md for implementors to know what they need to update to support the spec   version X.Y.Z would be useful.
But well yeah I guess I can just do tags and create that history.md anyway.
And tell people to git checkout tags/X.Y.Z

I guess it's not that important to have an actual package manager for this, idk.

**Comment by roblabla** [comment]
I think asking people to either use a git submodule (tagging the correct release) is reasonable enough for both this and mc-data.

**Comment by mhsjlw** [comment]
I should mention [bower](https://bower.io/) as well.
